src/main/java/com/xebialabs/deployit/ci/JenkinsDeploymentOptions.java

```diff
@@ -102,6 +102,7 @@ public FormValidation doCheckEnvironment(@QueryParameter(value = "credential") @
             @QueryParameter final String value,
             @AncestorInPath AbstractProject<?,?> project)
         {
+            Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);
```

Comment:
I am not convinced this is the correct permission, given that an `AbstractProject` would be `@AncestorInPath`.

Reply:
modified.
```
public FormValidation doCheckEnvironment(@QueryParameter(value = "credential") @RelativePath(value = "..") String credential,
            @QueryParameter(value = "credential") String credential2,
            @QueryParameter final String value,
            @AncestorInPath AbstractProject<?,?> project)
{
    project.checkPermission(Jenkins.ADMINISTER);
```